source/Modules/Controller/connection.js

```diff
@@ -0,0 +1,203 @@
+import qs from 'query-string';
+import extension from 'extensionizer';
+import ERRORS from '@background/errors';
+import PlugController from '@psychedelic/plug-controller';
+import { validatePrincipalId } from '@shared/utils/ids';
+import { CONNECTION_STATUS } from '@shared/constants/connectionStatus';
+import {
+  getApps,
+  setApps,
+  getApp,
+  removeApp
+} from '../storageManager';
+import SIZES from '../../Pages/Notification/components/Transfer/constants';
+
+export class ConnectionModule {
+  constructor(backgroundController, secureController, keyring) {
+    this.keyring = keyring;
+    this.secureController = secureController;
+    this.backgroundController = backgroundController;
+  }
+
+  // Utils
+  #getHandlerObjects() {
+    return [this.#isConnected(), this.#disconnect(), this.#requestConnect()];
+  }
+
+  #secureWrapper({ args, handlerObject }) {
```

Comment:
why the hashtag naming?

Reply:
Those are private functions, used solely inside the class